doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,722 @@
+    "In this tutorial, for simplicity, we will use the same particle types as the atomic number in ASE.\n",
+    "This means the mapping can simply be created by:\n",
+    "\n",
+    "```python\n",
+    "type_mapping = {x: x for x in set(atoms.numbers)}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for atom in ethanol_atoms:\n",
+    "    system.part.add(pos=atom.position, type=atom.number, mass=atom.mass)\n",
+    "system.ase = ASEInterface(type_mapping={x: x for x in set(ethanol_atoms.numbers)})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will use the [ZnDraw](https://github.com/zincware/ZnDraw) visualizer integrated with ESPResSo to follow the simulation. To see the effect of the geometry optimization, we display the atomic forces. You can either follow the simulation inside this Notebook or open the app in a dedicated window to the side, following the printed URL."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "vis = Visualizer(system)\n",
+    "vis.zndraw.config.scene.vectors = \"forces\"\n",
+    "vis.zndraw.config.scene.vector_scale = 5\n",
+    "vis.zndraw.config.scene.simulation_box = False\n",
+    "vis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To calculate the ground state, we can use the steepest descent algorithm of ESPResSo for the energy minimization."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "system.integrator.set_steepest_descent(f_max=0.1, gamma=4, max_displacement=0.001)  # Set the geometry optimisation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In practice this means that we use the MACE Model to predict the forces on the atoms, which are then provided to ESPResSo as external forces and ESPResSo takes care of the atom displacements."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "energies = []\n",
+    "for _ in tqdm.trange(100, ncols=120):\n",
+    "    atoms = system.ase.get()\n",
+    "    atoms.calc = mace_mp_calc\n",
+    "    system.part.all().ext_force = atoms.get_forces()\n",
+    "    energies.append(atoms.get_potential_energy())\n",
+    "    vis.zndraw.append(atoms)\n",
+    "    system.integrator.run(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Exercise:**\n",
+    "- Take a look at the energies and the molecule in the visualizer during the minimisation. \n",
+    "- How does the energy compare to the literature value of ~3341.86 eV for the ground state?"
```

Comment:
* this energy value needs a citation
* this energy value is not reproduced by the MACE-MP-0 model in the current form of the tutorial

Reply:
Energy is now resolved and correct.